doc/release-notes/8947-payara-update.md

```diff
@@ -0,0 +1,76 @@
+# Update to Payara 5.2022.3 highly recommended
+
+*NOTE: this might be rephrased to "required" depending on https://github.com/IQSS/dataverse/pull/8915 being merged.*
+
+With lots of bug and security fixes included, we encourage everyone to update to Payara 5.2022.3 as soon as possible.
+
+**Note:** with the approaching EOL for the Payara 5 Community release train it's likely we will switch to a
+yet-to-be-released Payara 6 in the not-so-far-away future.
+
+We recommend you ensure you followed all update instructions from the past releases regarding Payara.
+(latest Payara update was for [v5.6](https://github.com/IQSS/dataverse/release/v5.6))
+
+Upgrading requires a maintenance window and downtime. Please plan ahead, create backups of your database, etc.
+
+The steps below are a simple matter of reusing your existing domain directory with the new distribution.
+But we also recommend that you review the Payara upgrade instructions as it could be helpful during any troubleshooting:
+[Payara Release Notes](https://docs.payara.fish/community/docs/Release%20Notes/Release%20Notes%205.2022.3.html)
+
+If you are running Payara as a non-root user (and you should be!), remember not to execute the commands below as root.
+Use `sudo` to change to that user first. For example, `sudo -i -u dataverse` if `dataverse` is your dedicated
+application user.
+
+In the following commands we assume that Payara 5 is installed in `/usr/local/payara5`. If not, adjust as needed.
+
```

Comment:
```suggestion

Note that `lib/databases` now listed below alongside `generated and `osgi-cache` as directories that need to be cleared out. For more information, see #8064.

```

Reply:
It looks a bit duplicated. Should we remove the hint below the instruction? Maybe leave it in both places, but align the issue number? Maybe also emphasize this is not a regular thing to do as with the caches but is one shot for this update?

Reply:
Oh! I missed the hint below the instruction. Sorry!

I do wonder though... should we mention `lib/databases` somewhere in the guides? And about how blowing it away is not a regular thing to do, as you say? 😄

Reply:
I added it in the guides at some places where the others directories are present already.

Anywhere else?